test/integration/scheduler/queue_test.go

```diff
@@ -170,6 +172,108 @@ func TestServiceAffinityEnqueue(t *testing.T) {
 	}
 }
 
+func TestCoreResourceEnqueue(t *testing.T) {
+	cfg := configtesting.V1beta2ToInternalWithDefaults(t, v1beta2.KubeSchedulerConfiguration{
+		Profiles: []v1beta2.KubeSchedulerProfile{{
+			SchedulerName: pointer.StringPtr(v1.DefaultSchedulerName),
+			Plugins: &v1beta2.Plugins{
+				PreFilter: v1beta2.PluginSet{
+					Enabled: []v1beta2.Plugin{
+						{Name: noderesources.FitName},
+					},
+				},
+				Filter: v1beta2.PluginSet{
+					Enabled: []v1beta2.Plugin{
+						{Name: tainttoleration.Name},
+						{Name: noderesources.FitName},
+					},
+				},
+			},
```

Comment:
These plugins are enabled by default in v1beta2, and nothing to override here, so just remove those lines should be good.

Reply:
Aha, I modified from the original ServiceAffinity test.. Updated now.